database/src/main/java/com/firebase/ui/database/FirebaseListAdapter.java

```diff
@@ -51,10 +51,10 @@
  */
 public abstract class FirebaseListAdapter<T> extends BaseAdapter {
 
+    private Activity mActivity;
```

Comment:
I've had some cases in the past where I wanted to use the activity and layout in my concrete implementation. I'm not sure if these still apply, but would prefer to keep these protected for now.